theano/gpuarray/tests/run_dnn_conv.py

```diff
@@ -0,0 +1,153 @@
+# This script allows to run one specific cuDNN convolution test case.
+# python run_dnn_conv.py --help         # Print help.
+# python run_dnn_conv.py {fwd|bwd-filter|bwd-data} {2d|3d} -a <algo> -i <inputShape> -f <filterShape> ...
+
+from __future__ import absolute_import, print_function, division
+
+import argparse
+import sys
+
+import theano
+from theano.configdefaults import SUPPORTED_DNN_CONV_ALGO_RUNTIME
+from theano.gpuarray.cudnn_defs import (HALF, FLOAT, DOUBLE,
+                                        TRUE_HALF_CONFIG, PSEUDO_HALF_CONFIG, FLOAT_CONFIG, DOUBLE_CONFIG)
+from theano.gpuarray.tests.check_dnn_conv import (cudnn, TestDnnConv2D, TestDnnConv3D, CheckDnn)
+from theano.tensor.nnet.abstract_conv import get_conv_output_shape
+
+
+class TupleAction(argparse.Action):
+    # Tuple extractor for command line args parser.
+    def __call__(self, parser, namespace, values, option_string=None):
+        values = tuple(int(v) for v in values.split(','))
+        setattr(namespace, self.dest, values)
+
+
+class BorderAction(TupleAction):
+    # Border extractor for command line args parser.
+    def __call__(self, parser, namespace, values, option_string=None):
+        if values not in ('valid', 'full', 'half'):
+            super(BorderAction, self).__call__(parser, namespace, values, option_string)
+        else:
+            setattr(namespace, self.dest, values)
+
+
+if __name__ != '__main__':
```

Comment:
Move this before the class and add this information in the file docstring.

Reply:
Fixed